clang/unittests/Analysis/FlowSensitive/TransferTest.cpp

```diff
@@ -4099,6 +4099,20 @@ TEST(TransferTest, LoopDereferencingChangingRecordPointerConverges) {
   ASSERT_THAT_ERROR(checkDataflowWithNoopAnalysis(Code), llvm::Succeeded());
 }
 
+TEST(TransferTest, LoopWithDisjunctiveConditionConverges) {
```

Comment:
Test naming: I think the key aspect is not that the condition is a disjunction but that it's a short-circuited binary operator -- so maybe `LoopWithShortCircuitedConditionConverges`?